src/views/BondV2/ChooseBondV2.tsx

```diff
@@ -161,8 +161,8 @@ function ChooseBondV2() {
           <div className="help-text">
             <em>
               <Typography variant="body2">
-                Important: New bonds are auto-staked and no longer vest linearly. Simply claim as sOHM or gOHM at the
-                end of the term.
+                Important: New bonds are auto-staked (accrues rebase rewards) and no longer vest linearly. Simply claim
```

Comment:
this should say `...accrue rebase rewards` rather than `accrues rebase rewards` I think? 
bonds accrue rebase rewards.